Issue with chocolatey distribution

[chocolatey step](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650) fails with: 

```
Run choco apikey --key $Env:API_KEY --source https://push.chocolatey.org/
[2](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:2)
  choco apikey --key $Env:API_KEY --source https://push.chocolatey.org/
[3](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:3)
  choco push micronaut..nupkg --source https://push.chocolatey.org/ --force
[4](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:4)
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
[5](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:5)
  env:
[6](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:6)
    API_KEY: ***
[7](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:8)
Chocolatey v1.3.1
[8](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:9)
Added ApiKey for https://push.chocolatey.org/
[9](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:10)
Chocolatey v1.3.1
[10](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:11)
File specified is either not found or not a .nupkg file. 'micronaut..nupkg'
[11](https://github.com/micronaut-projects/micronaut-starter/actions/runs/4571849846/jobs/8070867650#step:6:12)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -378,6 +378,7 @@ jobs:
     name: Bump Chocolatey package
     runs-on: windows-latest
     needs: [windows]
+    shell: pwsh
     steps:
       - name: Determine default branch
         uses: octokit/request-action@v2.x
@@ -418,7 +419,7 @@ jobs:
           (Get-Content ./tools/chocolateyinstall.ps1) -replace '^\$version.*', "`$version = '$Version'" | Set-Content ./tools/chocolateyinstall.ps1
           (Get-Content ./tools/chocolateyinstall.ps1) -replace '  checksum\s+=.*', "  checksum      = '$SHA'" | Set-Content ./tools/chocolateyinstall.ps1
           Get-Content ./tools/chocolateyinstall.ps1
-          echo "nuget_version=$NugetVersion" >> $GITHUB_OUTPUT
+          echo "nuget_version=$NugetVersion" >> $env:GITHUB_OUTPUT
       - name: Create package
         working-directory: chocolatey
         run: choco pack

